crates/core/tedge_mapper/src/c8y_converter.rs

```diff
@@ -6,18 +7,23 @@ use c8y_smartrest::smartrest_serializer::{SmartRestSerializer, SmartRestSetSuppo
 use c8y_translator::json;
 use mqtt_client::{Message, Topic};
 use std::collections::HashSet;
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
 use thin_edge_json::alarm::ThinEdgeAlarm;
 
 const SMARTREST_PUBLISH_TOPIC: &str = "c8y/s/us";
+const INVENTORY_FRAGMENTS_FILE_LOCATION: &str = "/etc/tedge/devices/info.json";
```

Comment:
```suggestion
const INVENTORY_FRAGMENTS_FILE_LOCATION: &str = "/etc/tedge/device/extras.json";
```

Reply:
If we call it inventory internally, why not call that file also like that? Like "inventory.json"